cmd/katapult/config_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/krystal/katapult-cli/config"
+)
+
+const mockConfigFormat = `---
+api_key: %s
+api_url: %s
+
+`
+
+var mockConfigSuite = []struct {
+	apiKey string
+	apiURL string
+}{
+	{
+		apiKey: "",
+		apiURL: "",
+	},
+	{
+		apiKey: "test",
+		apiURL: "",
+	},
+	{
+		apiKey: "test",
+		apiURL: "test",
+	},
+	{
+		apiKey: "",
+		apiURL: "test",
+	},
+}
+
+func TestConfig(t *testing.T) {
+	conf, err := config.New()
+	if err != nil {
+		t.Fatal(err)
+	}
+	for _, v := range mockConfigSuite {
```

Comment:
Use Subtests here https://blog.golang.org/subtests

We tend to stick to using the variable name ``tt`` for the current test case executing.